src/views/docs-view/utils/get-back-to-link.ts

```diff
@@ -0,0 +1,54 @@
+import { SidebarBackToLinkProps } from 'components/sidebar/components/sidebar-back-to-link'
+import { ProductData, RootDocsPath } from 'types/products'
+
+/**
+ * Generate `backToLinkProps` for the current `rootDocsPath`,
+ * for use with `DocsView`.
+ *
+ * For nested rootDocsPaths, the `backToLinkProps` will
+ * lead to the "parent" rootDocsPath if there is one.
+ *
+ * For other cases, the `backToLinkProps` will lead to the product home page.
+ *
+ * A rootDocsPath is considered "nested" if its path contains a slash.
+ * If we have a "nested" rootDocsPath, we should try to find the parent
+ * rootDocsPath. If we can't find a "parent" rootDocsPath,
+ * or if this is not a "nested" rootDocsPath, then we should use
+ * a back-to "<product> Home" link.
+ */
+export function getBackToLink(
```

Comment:
Have leaned hard into the function description here to add context on why this feels necessary. Happy to expand further if anything is unclear 🙇‍♂️